dht/dhtcore/NodeStore.c

```diff
@@ -2442,3 +2442,17 @@ uint64_t NodeStore_timeSinceLastPing(struct NodeStore* nodeStore, struct Node_Tw
     }
     return now - lastSeen;
 }
+
+void NodeStore_destroyNode(struct NodeStore* nodeStore, uint8_t* addr)
```

Comment:
Name of this function is problematic, it indicates that the node is to be destroyed (unconditionally) but it's behaviour is NodeStore_destroyNodeIfNoKnownPath()